content/en/tracing/setup_overview/setup/php.md

```diff
@@ -317,6 +317,32 @@ To remove the PHP tracer:
 
 **Note**: If you are using second level caching in OPcache by setting the parameter `opcache.file_cache`, remove the cache folder.
 
+## Troubleshooting an application crash
+
+While very rare, it can happen that the PHP tracer causes an application to crash. Typically this happens because of a segmentation fault.
+When this happens, the best path ahead is to reach out to support providing a description of the problem and a core dump. Unless the procedure to recreate the application crash is trivial, without a core dump there is very little we can do to help.
+
+### Obtaining a core dump
+
+Obtaining a core dump for php applications can be tricky, especially on PHP-FPM. Here are a few tips to help you obtain a core dump.
+
+First things first: did the php-fpm generate any core dump at all? The answer to that question is in the application error log.
+
+An occurrence of `(SIGSEGV)` signals that the core was not dumped. For example: `WARNING: [pool www] child <pid> exited on signal 11 (SIGSEGV) after <duration> seconds from start`.
+
+On the other hand, an occurence of `(SIGSEGV - core dumped)` means it has been dumped. For example: `WARNING: [pool www] child N exited on signal 11 (SIGSEGV - core dumped) after X.Y seconds from start`.
+
+In order to know where the core dump is generated run `cat /proc/sys/kernel/core_pattern`. The default value is typically `core`, meaning that a file named `core` will be generated in the web root folder.
+
+Possible reasons causing a core dump not to be generated:
```

Comment:
Let's rephrase this as things they can DO to get it to generate. I'm am not 100% confident in some of my rephrasings that follow, so correct me if any are mistaken.
```suggestion
If no core dump was generated, check the following configurations and change them as needed:
```